### DISCUSSION TITLE
v2.0.0 updates

### DISCUSSION THREAD

**Discussion by gahabana** [MoleMan1024/audiowagon#69] [General]
Dear @MoleMan1024 , thank you for continuing to make (great) updates !!!

I am waiting for 2.2 Volvo AAOS to be installed (should be available OTA this week) and i do hope it will bring back USB file access (not sure )....

Many of us have huge libraries so one 'enhancement' (not sure how hard that would be) is ability to from playing screen or some other(s) add a song to 'new playlist' or 'existing playlist' ... i have 40,000 songs and I would love to be able when playing 'random' music and song i did not hear in a long time pops up - i would love to be able to add that to 'on the go' playlist .

Thank you very much !

**Comment by MoleMan1024** [comment]
hi, yeah unlikely that I will implement this I guess, would be lots of work to modify and manage playlists etc. I feel like there are enough tools out there to manage your playlists offline, don't think this is worth to implement in a car.
You can raise a feature request and if it gains enough traction I might reconsider.
